src/jwk.ts

```diff
@@ -70,7 +70,14 @@ type EcSignatureJwk = Jwk & {
   y: string;
 };
 
-export type SignatureJwk = RsaSignatureJwk | EcSignatureJwk;
+type OkpSignatureJwk = Jwk & {
```

Comment:
I had too lookup what OKP was at https://www.rfc-editor.org/rfc/rfc8037.html#page-3

Is it worth putting something in the comments for...
- Octet Key Pair (OKP)
- Edwards-curve Digital Signature Algorithm (EdDSA)

Reply:
Good point, added that now